doc/index.md

```diff
@@ -0,0 +1,39 @@
+# Not a Bot - Discord Bot
+
+Not a Bot is a french moderation bot designed for the [Not a Name](https://discord.gg/zcWp9sC).
+
+It is based on multiple configurable modules where each ones have their own task. 
+
+The bot is made in Lua, with a custom wrapper around the Discordia library.
+
+## Summary
+
+### Modules
+
+- [Ban]() : Ban management module
+- [Channel]() : Channel management module
+- [Game]() : 
```

Comment:
Game is the global module handling bot status

Reply:
I will recreate this PR, I made some mistakes.
(I will also update the Game description in the doc)